frontend/src/app/util/number.util.ts

```diff
@@ -0,0 +1,6 @@
+export class NumberUtil {
+
+  static rand = (max: number): number => Math.floor(Math.random() * max);
+
+  static range = (size: number, startFrom: number = 0) => Array.from(new Array(20), (x, i) => i + startFrom);
```

Comment:
why 20?

Reply:
No idea, was drunk maybe. Should be the size